qt-models/filtermodels.cpp

```diff
@@ -42,6 +42,7 @@ namespace {
 		if (tags.isEmpty())
 			return true;
 		QStringList dive_tags = get_taglist_string(d->tag_list).split(",");
+		dive_tags.append(divemode_text_ui[d->dc.divemode]);
```

Comment:
This should probably be translated by `gettextFromC::tr(divemode_text_ui[d->dc.divemode])`, for now. We can cache the translations later.

Reply:
Done.